src/user_interface/menu.rs

```diff
@@ -192,4 +251,6 @@ fn to_key_code(ch: char) -> Option<VirtualKeyCode> {
         'Z' => Z,
         _ => return None
     })
-}
\ No newline at end of file
+}
+
+const ALWAYS: fn(MenuEnvironment) -> bool = |_| true;
```

Comment:
Is there a reason this isn't just a function?

Reply:
I consider it to be a constant.